app/controllers/admin/creator_settings_controller.rb

```diff
@@ -1,15 +1,26 @@
 module Admin
   class CreatorSettingsController < Admin::ApplicationController
-    ALLOWED_PARAMS = %i[community_name logo_svg primary_brand_color_hex invite_only_mode public checked_code_of_conduct
-                        checked_terms_and_conditions].freeze
+    ALLOWED_PARAMS = %i[community_name primary_brand_color_hex invite_only_mode public checked_code_of_conduct
+                        checked_terms_and_conditions logo].freeze
 
-    def new; end
+    def new
+      @max_file_size = LogoUploader::MAX_FILE_SIZE
+      @logo_allowed_types = (LogoUploader::CONTENT_TYPE_ALLOWLIST +
+        LogoUploader::EXTENSION_ALLOWLIST.map { |extension| ".#{extension}" }).join(",")
```

Comment:
This constant gets added in this PR, so wouldn't it make sense to keep it in a format that doesn't require further transformation for its primary use case? This way it would also be more distinct from `IMAGE_TYPE_ALLOWLIST` which right now has the exact same content.

Reply:
@citizen428 can you clarify what you mean and which constant you're referring to? 

I suspect you're referring to 'EXTENSION_ALLOWLIST'? 

At the moment, I've formatted the allowlist constants in a way that Carrierwave and/or Carrierwave::Bombshelter expects these allowlists and then we've reused them for the validation on the front end.

Reply:
I see, if CarrierWave expects them this way then forget about it.